src/Module.Server/Common/Models/CrpgItemRequirementModel.cs

```diff
@@ -23,13 +23,26 @@ public static int ComputeItemRequirement(ItemObject item)
     }
 
     private static int ComputeCrossbowRequirement(ItemObject item)
+{
+    int strengthRequirementForTierTenCrossbow;
+
+    // Check if the item is a crossbow
+    if (item.ItemType != ItemObject.ItemTypeEnum.Crossbow)
     {
-        int strengthRequirementForTierTenCrossbow = 24; // Tiers are calulated in CrpgValueModel. 0<Tier=<10 . By design the best is always at Ten.
```

Comment:
we have validation of the `requirement` feature in [WebUi](https://github.com/crpg2/crpg/blob/0656550b8f5c525a104ed4798561ba240683657b/src/WebUI/src/services/characters-service.ts#L727).
I assume that this code will not affect `items.json` and there will still be `24 str` there?
Then we need to change the logic of `strengthRequirementForTierTenCrossbow` calculation in `ItemExporter`

Reply:
This change eventually needs to get to the items.json - just not sure how it gets there. How do I update the ItemExporter? Is it a branch or its own repo or?

Reply:
You know, everything seems fine.
[the value](https://github.com/crpg2/crpg/blob/master/src/Module.Client/DataExport/ItemExporter.cs#L46) is taken from your function

Reply:
> This change eventually needs to get to the items.json - just not sure how it gets there. How do I update the ItemExporter? Is it a branch or its own repo or?

We will discuss this tomorrow. But yes, the exporter is enabled in the branch `crafterharmonyexporter`. We could create a seperate module like was done with the crpg editor to make it easier.